tests/network/sriov.go

```diff
@@ -102,6 +103,7 @@ var _ = Describe("[Serial]SRIOV", func() {
 			var err error
 			vmi, err = createAndWaitVMI(vmi)
 			Expect(err).ToNot(HaveOccurred())
+			DeferCleanup(deleteVMI, vmi)
```

Comment:
Why do we need `DeferCleanup` ? why not just defer?
The first is context aware that is needed for example on `BeforeEach` (where it will auto create an `AfterEach` as you said)

Reply:
A simple `defer` will not allow the reporter to collect the state of the system once an error occurs in the test.